templates/contact.html

```diff
@@ -8,40 +8,43 @@
 <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js"></script>
 {% endblock head %}
 
-{% macro row(title, slug, email, people) %}
-<tr>
-  <td> <!--class="d-flex align-items-center gap-2">-->
-    <div class="fs-4 icon baseline"><img src="{{ get_url(path="assets/exec-icons/" ~ slug ~ ".svg") }}"></div>
-    {{ title }}
-  </td>
-  <td>
-    {{ people | join(sep=", ") }}
-  </td>
-  <td>
-      <a target="_blank" rel="noopener" href="mailto:{{ email }}"
-        aria-label="Email our {{ title }}">{{ email }}</a>
-  </td>
-</tr>
-{% endmacro %}
-
 {% block content %}
 
 {{ resource.content | safe}}
 
-<table class="table table-striped exec-table">
-  <thead>
-    <tr>
-      <th>Role</th>
-      <th>People</th>
-      <th>Contact</th>
-    </tr>
-  </thead>
-  <tbody>
-      {{ self::row(title="Exec", slug="all", email="exec@uwcs.co.uk", people=["All Exec"]) }}
-    {% for role in section.pages %}
-      {{ self::row(title=role.title, slug=role.slug, email=role.extra.email, people=role.extra.people) }}
-    {% endfor %}
-  </tbody>
-</table>
+{% for role in section.pages %}
+  <div class="role">
+    <div class="fs-4 icon baseline exec-title">
+      <img src="{{ get_url(path='assets/exec-icons/' ~ role.slug ~ '.svg') }}" alt="{{ role.title }}">
+      {% if role.extra.people | length > 1 %}
+        <h3>{{ role.extra.plural }}<h3>
+      {% else %}
+        <h3>{{ role.title }}<h3>
+      {% endif %}
+    </div>
+    <a href="mailto:{{role.extra.email}}" aria-label="Email our {{role.title}}">{{role.extra.email}}</a>
```

Comment:
IMO, place the roles emails with the people in the roles, even tho it leads to duplication (unless you have subheadings per role or something). At very least move this under the images.

This would also be the place you could list people's other socials if they want to: LinkedIn perhaps, I would say Discord but some people didn't like that last time it was brought up

Reply:
Perhaps then about/exec is a better place for exec photos?